config/apidoc.php

```diff
@@ -9,6 +9,11 @@
      */
     'type' => 'static',
 
+    /*
+     * Static output folder: Uncomment and change it if you need the documentation to be generated in a different folder
+     */
+    // 'output_folder' => 'public/docs',
```

Comment:
The Laravel convention is not to comment out unused configs, but to set a reasonable default.

```suggestion
   'output_folder' => 'public/docs',
```